.travis/preinstall.sh

```diff
@@ -22,22 +23,12 @@ CC=/usr/bin/gcc CXX=/usr/bin/g++ cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DALLOC
 CC=/usr/bin/gcc CXX=/usr/bin/g++ make
 sudo make install
 
-# build and install cryptopp
-cd /tmp
-git clone https://github.com/weidai11/cryptopp.git
-cd cryptopp
-git checkout CRYPTOPP_8_2_0
-make
-sudo make install
-
-# build and install boost
-cd /tmp
-wget --https-only https://dl.bintray.com/boostorg/release/1.64.0/source/boost_1_64_0.tar.gz
-tar -xf boost_1_64_0.tar.gz
-cd boost_1_64_0
-./bootstrap.sh --with-libraries=system,filesystem
-./b2
-sudo ./b2 install
+# Install Conan package manager
+python3 -m pip install --upgrade wheel
+python3 -m pip install --upgrade conan
+source ~/.profile
```

Comment:
why do we need this?

Reply:
1. Travis needs Conan prebuilt package called wheel, for Conan installation.
2. When Conan is installed with pip install, usually a new directory is created for it. However, the directory is not appended automatically to the PATH and the Conan commands do not work. This can usually be solved restarting the session of the terminal or running "source ~/.profile"

Reply:
And who adds this directory to PATH?